SKPhotoBrowser/SKMesurement.swift

```diff
@@ -28,7 +28,8 @@ struct SKMesurement {
         return screenWidth / screenHeight
     }
     static var isPhoneX: Bool {
-        if isPhone && UIScreen.main.nativeBounds.height == 2436 {
+        let iPhoneXHeights: [CGFloat] = [2436, 2688, 1792]
```

Comment:
Jez🤦🏻‍♂️

Reply:
Why am I still subscribed to this repo? 😁
LG nach Wien 🙇‍♂️

Reply:
ok dh sobald ein neues Gerät mit einer anderen Auflösung rauskommt funktioniert das wieder nicht? 🤔

Reply:
Haha why can you even view this PR🙊 Your a not listed in the team of willhaben settings anymore🙈

Reply:
I guess anyone can view ;) It's not a private repo

Reply:
@mathaeus everthing is ready for your comeback ;)